pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -0,0 +1,434 @@
+                    <div class="form-group mb-0">
+                      <input type="hidden" name="__action" value="copy_assessment_groups">
+                      <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                      <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                      <button type="submit" class="btn btn-primary">Copy</button>
+                    </div>
+                  </div>
+              </div>
+              </form>
+            </div>
+          </div>
+        </div>
+        <div class="modal fade" id="addGroupModal" tabindex="-1" role="dialog" aria-labelledby="addGroupModalLabel">
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h1 class="h4 modal-title" id="addGroupModalLabel">Add a group</h1>
+              </div>
+              <form name="add-Group-form" method="POST">
+                <div class="modal-body">
+                  <div class="form-name">
+                    <label for="formName">Group Name</label>
+                    <input type="text" class="form-control" id="formName" name="groupname">
+                  </div>
+                  <br>
+                  <div class="form-uids">
+                    <label for="formUids">UIDs (Seperate with ",". Please make sure they are not in any other
+                      groups)</label>
+                    <input type="text" class="form-control" id="formUids" name="uids"
+                      placeholder="one@example.com, two@example.com, three@example.com">
+                  </div>
+                </div>
+                <div class="modal-footer">
+                  <input type="hidden" name="__action" value="addGroup">
+                  <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                  <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                  <button type="submit" class="btn btn-primary">Add</button>
+                </div>
+              </form>
+            </div>
+          </div>
+        </div>
+        <div class="modal fade" id="configModal" tabindex="-1" role="dialog" aria-labelledby="configModalLabel">
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h1 class="h4 modal-title" id="configModalLabel">Group Config</h1>
+              </div>
+              <form name="config-Group-form" method="POST">
+                <div class="modal-body">
+                  <span class="badge badge-secondary">id</span> <%= config_info.id %><br>
+                  <span class="badge badge-secondary">course_instance_id</span>
+                  <%= config_info.course_instance_id %><br>
+                  <span class="badge badge-secondary">minimum size</span> <%= config_info.minimum %><br>
```

Comment:
Why is this information duplicated here and below in the inputs?

Reply:
In case the instructor want to check the current group settings for his/her class